### PR TITLE
Fix PHPStan errors level 1

### PR DESCRIPTION
**Pull request by Art4**
This PR fixes all errors for PHPStan level 1.

Requires:

- [x] #1369

### CODE REVIEW

**Comment by Art4** [comment]
This PR is ready for review, but requires #1369. 

The messages.po is updated in https://github.com/friendica/friendica/pull/14581

**Comment by MrPetovan** [comment]
https://github.com/friendica/friendica-addons/pull/1369 has been merged, please rebase so that I can see the commits of this branch better.

**Comment by Art4** [comment]
> #1369 has been merged, please rebase so that I can see the commits of this branch better.

@MrPetovan Done.

File: mailstream/phpmailer/class.phpmailer.php [hunk 1]
```diff
@@ -3287,7 +3287,7 @@ protected function serverHostname()
         $result = 'localhost.localdomain';
         if (!empty($this->Hostname)) {
             $result = $this->Hostname;
-        } elseif (isset($_SERVER) and array_key_exists('SERVER_NAME', $_SERVER) and !empty($_SERVER['SERVER_NAME'])) {
+        } elseif (array_key_exists('SERVER_NAME', $_SERVER) and !empty($_SERVER['SERVER_NAME'])) {
```
Comment (MrPetovan) [inline mailstream/phpmailer/class.phpmailer.php]: This can be simplified further.
```suggestion
        } elseif (!empty($_SERVER['SERVER_NAME'])) {
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: mailstream/phpmailer/class.phpmailer.php (hunk 1 above)
Comment (Art4) [reply mailstream/phpmailer/class.phpmailer.php]: The phpmailer is a vendor class and I tried to change it a little at possible. It is also possible that the `SERVER_NAME` key does not exists, e.g. in command line mode.

Comment (MrPetovan) [reply mailstream/phpmailer/class.phpmailer.php]: I know, and it's fine, `empty()` takes care of missing array keys as well, just like `unset()`.

**Review by MrPetovan — APPROVED** [review]

**Comment by MrPetovan** [comment]
Moved to https://git.friendi.ca/friendica/friendica-addons/pulls/1579

**Pull request closed without merging**